res/controllers/Traktor-Kontrol-S4-MK3.js

```diff
@@ -391,6 +526,9 @@ class ComponentContainer extends Component {
     }
     reconnectComponents(callback) {
         for (const component of this) {
+            if (typeof component.unshift === "function" && component.unshift.length === 0) {
+                component.unshift();
+            }
```

Comment:
why is this dupe needed?
isn't it identical to that a few lines below?

Reply:
Hehe..
sure, remove and add a comment.

Unfortunately I don't really find time right now to test this thoroughly.
Will do tomorrow.